js/tests/unit/modal.js

```diff
@@ -86,6 +97,22 @@ $(function () {
       .bootstrapModal('show')
   })
 
+  QUnit.test('should set aria-hidden to true when hide is called', function (assert) {
+    var done = assert.async()
+
+    $('<div id="modal-test"/>')
+      .on('shown.bs.modal', function () {
+        assert.ok($('#modal-test').is(':visible'), 'modal visible')
+        assert.notEqual($('#modal-test').length, 0, 'modal inserted into dom')
```

Comment:
That does make more sense. Honestly that first part was just a copy from the 'should toggle when toggle is called' test